### PR TITLE
Add procedures for syncing to the stable release

### PR DESCRIPTION
**Pull request by kmoscoe**

### CODE REVIEW

File: custom_dc/custom_ui.md [hunk 1]
```diff
@@ -80,4 +80,4 @@ Alternatively, if you have existing existing CSS and Javascript files, put them
 
 See [`server/templates/custom_dc/custom/new.html`](https://github.com/datacommonsorg/website/blob/master/server/templates/custom_dc/custom/new.html) as an example.
 
-Note: Currently, making changes to any of the files in the `static/` directory requires that you rebuild a local version of the repo to pick up the changes, as described in [Build and run a local repo](/custom_dc/build_repo.html). We plan to fix this in the near future.
+Note: Currently, making changes to any of the files in the `static/` directory requires that you rebuild a local version of the repo to pick up the changes, as described in [Build a local image](manage_repo.md#build-repo). We plan to fix this in the near future.
```
Comment (keyurva) [inline custom_dc/custom_ui.md:83]: > We plan to fix this in the near future.

To deploy to the cloud they will always need to build it. Perhaps we can remove this line?

File: custom_dc/manage_repo.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+
+## Use a prebuilt image
+
+While you are just testing out data changes, you don't need to build the website, but can just use a prebuilt Data Commons image.
+
+Data Commons provides two prebuilt images in the Google Artifact Registry that you can download to run in a Docker container:
+
+- `gcr.io/datcom-ci/datacommons-website-compose:stable`. This is a tested, stable version but may be several weeks old.
+- `gcr.io/datcom-ci/datacommons-website-compose:latest`. This is the latest built version that is running in production.
```
Comment (keyurva) [inline custom_dc/manage_repo.md:22]: > running in production

That is not true. It just is the latest that was built from head.

File: custom_dc/manage_repo.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+   ```
+   cd website
+   ```
+1. Create a new branch synced to the stable release:
+
+   <pre>
+   git checkout -b <var>BRANCH_NAME</var> customdc_stable
+   </pre>
+
+1. To verify that your local repo is at the same version of the vode, run the following command:
```
Comment (keyurva) [inline custom_dc/manage_repo.md:67]: s/vode/code

File: custom_dc/manage_repo.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+- You are ready to deploy your custom site to GCP
+
+Rather than building from "head", that is, the very latest changes in Github, which may not have been tested, we recommend that you use a tested "release", with the tag `customdc_stable`, available at [https://github.com/datacommonsorg/website/releases/tag/customdc_stable](https://github.com/datacommonsorg/website/releases/tag/customdc_stable).
+
+Note: If you are working on a large-scale customization, we recommend that you use a version control system to manage your code. We provide procedures for Github, and assume the following:
+- You have a Github account and project.
+- You have created a fork off the main Data Commons `website` repo (https://github.com/datacommonsorg/website) and a remote that points to it, and that you will push to that fork. 
+
+
+### Sync a local workspace to the stable release
```
Comment (keyurva) [inline custom_dc/manage_repo.md:44]: Thanks for figuring it out and documenting it!

**Review by keyurva — APPROVED** [review]
Thanks Kara!

File: custom_dc/custom_ui.md (hunk 1 above)
Comment (kmoscoe) [reply custom_dc/custom_ui.md:83]: Yes, but this is specifically about testing local changes. I'll add, "even locally"

File: custom_dc/manage_repo.md (hunk 2 above)
Comment (kmoscoe) [reply custom_dc/manage_repo.md:22]: What?  Really? So it is actually less stable than customdc_release?

File: custom_dc/manage_repo.md (hunk 3 above)
Comment (kmoscoe) [reply custom_dc/manage_repo.md:67]: Gah! Fixed.

File: custom_dc/manage_repo.md (hunk 4 above)
Comment (kmoscoe) [reply custom_dc/manage_repo.md:44]: Thanks, it took me a while!

File: custom_dc/manage_repo.md (hunk 2 above)
Comment (kmoscoe) [reply custom_dc/manage_repo.md:22]: Also, what is the difference between the stable Docker image vs. the customdc_stable release then?

Comment (keyurva) [reply custom_dc/manage_repo.md:22]: The stable docker image will be based off of the customdc_stable release. We'll make it part of the oncall in the near future.

Comment (kmoscoe) [reply custom_dc/manage_repo.md:22]: Ah, OK. Rewritten, PTAL

Comment (keyurva) [reply custom_dc/manage_repo.md]: LG.

**Pull request merged**